tests/providers/aws/aws_cli.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Call libcloudforensics CLI tool for e2e testing."""
+import subprocess
+from typing import List, Optional
+
+from libcloudforensics import logging_utils
+from libcloudforensics.errors import ResourceCreationError
+from libcloudforensics.errors import ResourceNotFoundError
+from libcloudforensics.providers.aws.internal import ebs
+from libcloudforensics.providers.aws.internal import ec2
+from libcloudforensics.providers.aws.internal.account import AWSAccount
+
+logging_utils.SetUpLogger(__name__)
+logger = logging_utils.GetLogger(__name__)
+
+
+class AWSCLI:
+  """AWSCLI calls the libcloudforensics CLI tool for AWS operations."""
+  def __init__(self, aws: AWSAccount) -> None:
+    """Initialize the CLI class.
+
+    Attributes:
+      aws (AWSAccount): The AWS account to work with.
+    """
+    self.aws = aws
+
+  def StartAnalysisVm(
```

Comment:
Just for clarity, can we rename this to "RunStartAnalysisVm" command? And adjust docstrings to emphasize that this is just a wrapper around the CLI?